Kernel_23/include/CGAL/Point_3.h

```diff
@@ -79,6 +79,13 @@ class Point_3 : public R_::Kernel_base::Point_3
     : Rep(typename R::Construct_point_3()(Return_base_tag(), x, y, z))
   {}
 
+  template < typename T1, typename T2, typename T3 >
+  Point_3(T1&& x, T2&& y, T3&& z)
+    : Rep(typename R::Construct_point_3()(Return_base_tag(), std::forward<T1>(x),
+                                                             std::forward<T2>(y),
+                                                             std::forward<T3>(z)))
+  {}
```

Comment:
Here it is a perfect forwarding.

For what type of `FT` do we have move-semantic?

Reply:
Most of the bignums have a move constructor : mpq_class, mpq_rational, cpp_rational, Lazy_exact_nt, Mpzf, etc. The ref-counted ones like Gmpq have a move assignment but no (useful) move constructor.